packages/calypso-build/package.json

```diff
@@ -1,6 +1,6 @@
 {
 	"name": "@automattic/calypso-build",
-	"version": "2.0.0",
+	"version": "2.1.0",
```

Comment:
https://semver.org/ says:

> Given a version number MAJOR.MINOR.PATCH, increment the:
>
> 1. MAJOR version when you make incompatible API changes,
> 2. MINOR version when you add functionality in a backwards-compatible manner, and
> 3. PATCH version when you make backwards-compatible bug fixes.

We're not touching the API here, and I think we are backwards-compat (worst thing that can happen is double transpilation of some stuff?)